labgrid/remote/exporter.py

```diff
@@ -201,6 +201,7 @@ def __del__(self):
     def _get_start_params(self):
         return {
             'path': self.local.port,
+            'speed': self.local.speed,
```

Comment:
This shouldn't be required and does not change anything. The `start_params` are used to find out if the underlying path was changes i.e. because of an USB unplug & plug action by the user. The speed param is set in the configuration file and will never change, please remove this line.